frappe/public/js/frappe/form/multi_select_dialog.js

```diff
@@ -114,6 +71,10 @@ frappe.ui.form.MultiSelectDialog = Class.extend({
 			}
 		});
 
+		if (this.add_filters_group) {
+			this.make_filter_area();
+		}
```

Comment:
For your use case, you don't really need custom filters right now. Let's not add custom filters here, let's use standard filters only. 

We can remove this

Reply:
As talked discussed in the call there is a valid use case